draco/core/containers.py

```diff
@@ -1315,6 +1315,65 @@ class SpectroscopicCatalog(SourceCatalog):
     }
 
 
+class FormedBeam(ContainerBase):
+    """Container for formed beams.
+    """
+    
+    _axes = ('object_id', 'pol', 'freq')
+
+    _dataset_spec = {
+        'fbeam': {
```

Comment:
Any reason to not call it `beam`?

Reply:
Not a problem here. I'll make the change.
(In BeamForm I use the word `beam` to denote the primary beam returned by my Gaussian functions. So I use formed_beam to differentiate.)